docs/basics/kda/what-is-kda.md

```diff
@@ -27,14 +27,12 @@ The total supply is fixed at 1 billion tokens to be mined over 120 years
 
 ## What is the total number of tokens in circulation?
 
-See [Token Economic Model](https://medium.com/kadena-io/the-kadena-token-economic-model-8090d7545eef) for more detailed information, and recent update [here](https://medium.com/kadena-io/update-to-the-kadena-token-economic-model-21e1ec18f099).
+Current token circulation is about 185 million coins
```

Comment:
It's probably best to point to the explorer here as this value will be outdated pretty fast.